documentation/staging/content/userguide/managing-operators/using-helm.md

```diff
@@ -393,39 +389,24 @@ This value is required if `domainNamespaceSelectionStrategy` is `RegExp` and ign
 
 - The regular expression functionality included with Helm is restricted to linear time constructs and,
   in particular, does not support lookarounds. The operator, written in Java, supports these
-  complicated expressions. If you need to use a complex regular expression, then either set
-  `enableClusterRoleBinding` to `true` or create the necessary RoleBindings outside of Helm.
+  complicated expressions. If you need to use a complex regular expression, then either:
+     - Set `enableClusterRoleBinding` to `true`.
+        - When `enableClusterRoleBinding` is `false`  and namespaces are determined by list or regular expression,
+          then the Helm chart has to iterate all of the namespaces and determine which ones match.
+        - If you set `enableClusterRoleBinding` to `true` when selecting namespaces by list or regular expression,
+          then the Helm chart doesn't need to do anything special per namespace.
+     - Or, create the necessary RoleBindings outside of Helm.
 - If your operator Helm `enableClusterRoleBinding` configuration value is `false`, then
   then a running operator will _not_ have privilege to manage a newly added namespace
   that matches its regular expression until you upgrade
   the operator's Helm release.
   See [Ensuring the operator has permission to manage a namespace]({{< relref "/userguide/managing-operators/namespace-management#ensuring-the-operator-has-permission-to-manage-a-namespace" >}}).
-- If the deprecated `dedicated` setting is set to `true`,
-  then this value is ignored and
-  the operator will manage only domains in its own namespace.
-
-##### `dedicated` ***(Deprecated)***
```

Comment:
IMO, it is only appropriate to remove `dedicated` if Operator 4.0 is not going to support it. Is that the case?

Reply:
Yes.

Reply:
@tbarnes-us Rosemary is doing what I asked her to do.  This parameter has been deprecated for a long time now, we should remove it completely in 4.0.

Reply:
Rosemary filled me in above. Thanks.